### ISSUE TITLE
Fix ZIM file sizes on the download page

### ISSUE THREAD

**Issue by mhutti1** [kiwix/web#30]
ZIM files sizes are listed as considerably smaller than they are on http://www.kiwix.org/downloads. Probably because they are out of date.

**Comment by Popolechien** [comment]
I've started to fix the numbers based on http://download.kiwix.org/zim/ but I think we've got a broader problem of consistency: the zip file for WikiSource is 14.9 Gb, while the zim alone is 13.2 Gb. Is this because zim and zip aren't updated synchronously?

The generic http://download.kiwix.org/zim/wikisource_en_all.zim link points to wikisource_en_all_novid_2018-03.zim rather than wikisource_en_all_2017-02.zim - so admittedly the shortcut points to the latest zim file, not consistently to _all_ or _all_novid : this is fine but since we have way too many files in the /zim directory it would be an argument to discontinue either novid or all). 
@kelson42 thoughts?

Also last but not least we're reworking the website so ultimately this page will change. Now would be a good time to input suggestions.
